types/insane/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "private": true,
+    "name": "@types/insane",
+    "version": "1.0.9999",
+    "projects": [
+        "https://github.com/bevacqua/insane"
+    ],
+    "devDependencies": {
+        "@types/insane": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "Nicolás Bevacqua",
+            "githubUsername": "bevacqua"
```

Comment:
This should be you, not the package author

Reply:
Done